### ISSUE TITLE
Why only train set is used for cross validation and test set is left out in pycaret?

### ISSUE THREAD

**Issue by vickaryan** [pycaret/pycaret#1696]
I have a doubt why 10-fold cross validation is done only on train set because we generally need test set also for cross validation?

_Originally posted by @vickaryan in https://github.com/pycaret/pycaret/discussions/1689_

**Comment by vickaryan** [comment]
For training ML models, we either split data into train/test or do cross validation on the complete dataset as splitting method is considered inefficient. So, why in pycaret, all of the evaluation metrics are cross-validated results based on training set (70%) only. Could anyone please clear why test set is not included in cross validation process during training models and evaluating metrics?
